bin/bokeh-export.py

```diff
@@ -39,8 +40,10 @@
 fieldvalue = args.value
 
 if args.dest is None:
-    destpath = "."
+    destpath = "./"
 else:
+    if not os.path.exists(args.dest):
+        os.makedirs(args.dest)
     destpath = args.dest
 
 redisKeyMonth = "chp-5577-1:"+str(date)
```

Comment:
Maybe the initial source key (sensor) should be set as a configuration? via an option at the argument level and if not set, to set it as potiron alone.